src/Rule.ts

```diff
@@ -1,11 +1,13 @@
 import { Schema } from "extract-pg-schema";
 
-export type Reporter = (p: {
+export type Issue = {
```

Comment:
I named this object `Issue`. Doesn't that sound strange? Another idea is `Problem`.